api/cards/updateCards.spec.ts

```diff
@@ -0,0 +1,31 @@
+import * as httpMocks from "node-mocks-http";
+import "../../utils/cards";
+import updateCardsController from "./updateCards";
+
+const mockUpdateCardsStorage = jest.fn(() => Promise.resolve("updateInfo"));
+
+jest.mock("../../utils/cards", () => ({
+  updateCardsStorage: () => mockUpdateCardsStorage()
```

Comment:
Is it ok to replace with `updateCardsStorage: mockUpdateCardsStorage` or you don't want to pass parameters explicitly?

Reply:
You're right from js-perspective. But in case of jest you'll get `mockUpdateCardsStorage is undefined`
Thanks jest-specific trick